integration-tests/src/test/java/integration/IntegrationTestsConfig.java

```diff
@@ -0,0 +1,75 @@
+/**
+ * This file is part of Graylog.
+ *
+ * Graylog is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * Graylog is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with Graylog.  If not, see <http://www.gnu.org/licenses/>.
+ */
+package integration;
+
+import org.apache.http.client.utils.URIBuilder;
+
+import java.net.MalformedURLException;
+import java.net.URISyntaxException;
+import java.net.URL;
+
+public class IntegrationTestsConfig {
+    private static final String glBaseURi = System.getProperty("gl.baseuri", "http://localhost:12900");
```

Comment:
IMHO (and according to the [JLS](https://docs.oracle.com/javase/specs/jls/se8/html/jls-6.html)), constants should be uppercase. This also matches (most of) the rest of our codebase.